builtin/providers/github/resource_github_team.go

```diff
@@ -43,31 +45,29 @@ func resourceGithubTeamCreate(d *schema.ResourceData, meta interface{}) error {
 func resourceGithubTeamRead(d *schema.ResourceData, meta interface{}) error {
 	client := meta.(*GithubClient).client
 
-	_, err := getGithubTeam(d, client)
+	t, err := getGithubTeam(d, client)
 	if err != nil {
 		d.SetId("")
 		return nil
 	}
-	//TODO need to update Description but go-github doesn't expose it
+	d.Set("name", t.Name)
+	d.Set("description", t.Description)
 	return nil
 }
 
 func resourceGithubTeamUpdate(d *schema.ResourceData, meta interface{}) error {
 	client := meta.(*GithubClient).client
-	team, err := getGithubTeam(d, client)
-
-	if err != nil {
-		d.SetId("")
-		return nil
-	}
-
-	//TODO need to set Description on team before sending it, but go-github doesn't expose it
+	n := d.Get("name").(string)
+	s := d.Get("description").(string)
+	id := d.Id()
 
-	team, _, err = client.Organizations.EditTeam(*team.ID, team)
+	_, _, err := client.Organizations.EditTeam(toGithubId(id), &github.Team{
+		Name:        &n,
+		Description: &s,
```

Comment:
You don't need to create a whole new `&github.Team{}` here...you can just update the values on `team` and pass it back in.